queries/julia/highlights.scm

```diff
@@ -168,17 +172,19 @@
 (export_statement
   ["export"] @include)
 
-[
-  "using"
-  "module"
-  "import"
-] @include
+(module_definition
+  ["module" "end"] @namespace)
```

Comment:
namespace is usually for namespace identifiers

Reply:
Ok. would
```
(module_definition
  "module" @include
  name: (identifier) @namespace
  "end" @include)
```
make more sense in this case?

Reply:
for the identifer yes, but `include` is usually only used for keywords importing a module from another file

Reply:
> Highlighting profits from a more flexible approach

No, users will just get unexpected highlights. If more flexibility is need, usually a new capture for the new thing is better as some people get really frustrated when they can't change a "wrong" highlight (`@keyword.module` with default `@keyword` highlight which could also apply for Ruby modules). From the experience with this repo, you really need strong reasons to give a distinct highlight to keywords. `module`/`end` might be conceivable for default `include` though

Reply:
> include is usually only used for keywords importing a module from another file

You're right. I was comparing with the rust queries for reference, but I forgot `mod` does both (declaration or import). Although I also agree with @clason we can be more lenient with some aspects of the highlighting.

Also, on second thought I won't write the `@namespace` query; simply because in practice, module namespacing is used very little in Julia and there aren't any other `@namespace` queries that relate to it.

Reply:
Yes, it's a similar concept. It would be no problem to use include here especially when Rust already uses `mod` in the same way. Just have a look whether it fits into general look and feel.